### ISSUE TITLE
Problem in Unity 5.5

### ISSUE THREAD

**Issue by zoeesilcock** [lite3/Unity-BitmapFontImporter#4]
As mentioned in the comments on the Asset Store there is a problem that occurs after updating to Unity 5.5. At first I thought something had changed in Unity which broke the import entirely. After importing a font it does not render at all. Fonts that had been imported with an older version of Unity work fine until you re-import them. In the process of checking for changes in the generated files I found that there was no change after re-importing them. Then I stumbled upon the fact that closing and re-opening the Unity project fixed the problem.

So bottom line is that the plugin does still work in Unity 5.5 but it requires a restart after importing fonts. With this in mind it might be possible to reload some assets programmatically after importing.

**Comment by litefeel** [comment]
@zoeesilcock 
Thinks for your feedback. now I solved this problem on branch dev, I will submit it to Asset Store later.

**Comment by litefeel** [comment]
Fixed it in v1.5
